### ISSUE TITLE
Spladder Prep Yields No Results And No Error Without `--qmode`

### ISSUE THREAD

**Issue by yr542** [ratschlab/spladder#202]
* Spladder Version: Regrettably I cannot find this (though I recently installed it)
* Python Version: 3.11.5
* Operating System: Linux

## Description

I saw no indication in the documentation but spladder prep is found in the package so I tried to use it in  the following:

### Command & Error:

**My commands:**

```
cd /path/to/the/directory/with/mycontrol/bams/
spladder prep \
    --bams "C-1.bam,C-2.bam,C-3.bam" \
    --annotation "/the/path/on/my/HPC/to/the/reference/annotation.gtf" \
    --qmode collect \
    --verbose
```

**My error:**

```
usage: spladder [-h] {prep, build, test, viz} ...
spladder: error: unrecognized arguments: --qmode collect
usage: spladder [-h] {prep, build, test, viz} ...
spladder: error: unrecognized arguments: --qmode collect
```

I also tried without the `qmode` command and it runs with no errors/warning but I do not get an output. Is the prep step unnecessary - what am I doing incorrectly?

**Comment by akahles** [comment]
Thanks @yr542 for reaching out. The `prep` mode is not part of the "official" SplAdder modes. However, I should add it to the documentation. There is a few dependencies that SplAdder relies on (e.g., a pre-parsing of the annotation file or the sparsification of bam files if requested). When running SplAdder in parallel on the same resources, this can lead to race conditions. Thus, in a workflow, we first run the `prep` mode to create these dependencies and then subsequently run the graph jobs in parallel. 

Consequently, only a small number of options are defined for the `prep` mode. `--qmode collect` is only defined for `build`.
